roxie/ccd/ccdcontext.cpp

```diff
@@ -1775,8 +1776,9 @@ class CRoxieContextBase : implements IRoxieAgentContext, implements ICodeContext
     }
     virtual IEngineContext *queryEngineContext() { return NULL; }
     virtual char *getDaliServers() { throwUnexpected(); }
-    virtual unsigned getWorkflowId() { return 0; } // this is a virtual which is implemented in IGlobalContext
-
+    // The following method isn't used.  Renaming and leaving here to avoid changing interface
+    // (Note, this getWorkflowId doesn't implement virtual from any derived classess.)
+    virtual unsigned getWorkflowIdDeprecated() { throwUnexpected(); }
```

Comment:
why is this here (+ not overridden)? This class (CRoxieContextBase) does not implement IGlobalCodeContext